src/averageAges.js

```diff
@@ -20,6 +20,16 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const filterMan = !century
+    ? people.filter(human => human.sex === 'm')
+    : people.filter(human =>
+      Math.ceil(human.died / 100) === century && human.sex === 'm');
```

Comment:
In both cases you used filter, so it can be moved out of ternary operator (just use it inside for condition to filter)